### PR TITLE
[Backport-14282-stable8]

### PR DESCRIPTION
**Pull request by DeepDiver1975**
refs #14282

### CODE REVIEW

File: lib/private/updater.php [hunk 1]
```diff
@@ -21,29 +24,27 @@
...
+	/** @var bool */
 	private $updateStepEnabled;
 
 	/**
+	 * @param HTTPHelper $httpHelper
+	 * @param IConfig $config
 	 * @param \OC\Log $log
 	 */
-	public function __construct($httpHelper, $config,  $log = null) {
+	public function __construct(HTTPHelper $httpHelper, IConfig $config, $log = null) {
```
Comment (DeepDiver1975) [inline lib/private/updater.php]: Bääh - this will break the updater app :imp:

**Comment by karlitschek** [comment]
:+1:

File: lib/private/updater.php (hunk 1 above)
Comment (DeepDiver1975) [inline lib/private/updater.php]: > Bääh - this will break the updater app :imp:

https://github.com/owncloud/updater/pull/81

**Comment by PVince81** [comment]
Update worked, 3rd party apps were disabled :+1:

**Pull request merged**